go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/celestiaorg/knuu v0.16.1
 	github.com/celestiaorg/nmt v0.22.2
 	github.com/celestiaorg/rsmt2d v0.14.0
-	github.com/cometbft/cometbft-db v0.7.0
+	github.com/cometbft/cometbft-db v1.0.1
```

Comment:
Looks like the only breaking change in v1.0.0 got reverted in v1.0.1

https://github.com/cometbft/cometbft-db/blob/v1.0.1/CHANGELOG.md#v101